lib/stats.js

```diff
@@ -67,7 +67,7 @@ var stats = {
       var childrens = options.childrens ? stat.cutime + stat.cstime : 0
       var total = 0;
 
-      if(history.utime) {
+      if(typeof history.utime !== 'undefined') {
```

Comment:
Yes that's my proposal
Le mer. 29 juin 2016 à 17:42, Aaron Tarter <notifications@github.com> a
écrit :

> In lib/stats.js
> <https://github.com/soyuka/pidusage/pull/21#discussion_r68971950>:
>
> > @@ -67,7 +67,7 @@ var stats = {
> >        var childrens = options.childrens ? stat.cutime + stat.cstime : 0
> >        var total = 0;
> >
> > -      if(history.utime) {
> > +      if(typeof history.utime !== 'undefined') {
>
> If you dont change that if statement, history.utime=0 will still go in the
> else. Are you proposing that the if/else statement be removed altogether
> and replaced with that new assignment? I think that would work.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/soyuka/pidusage/pull/21/files/8490ad1dd1476849930ddc56458a71e438faf0df#r68971950>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABQr84GCviUgNRhlAxr3J5FJdsXoYlOhks5qQpKAgaJpZM4JAZYT>
> .
>